src/commands/layout/get.ts

```diff
@@ -0,0 +1,103 @@
+import { Args, Flags, ux } from "@oclif/core";
+
+import * as ApiV1 from "@/lib/api-v1";
+import BaseCommand from "@/lib/base-command";
+import { formatDate } from "@/lib/helpers/date";
+import { withSpinner } from "@/lib/helpers/request";
+
+export default class EmailLayoutGet extends BaseCommand<typeof EmailLayoutGet> {
+  static summary = "Display a single email layout from an environment.";
+
```

Comment:
Also let's add aliases here for this command as well i.e. `email-layout:get`, and `email_layout:get`